sysconfig/cloudinit.go

```diff
@@ -221,7 +223,12 @@ func CloudInitStatus() (CloudInitState, error) {
 		return CloudInitDisabledPermanently, nil
 	}
 
-	out, err := exec.Command("cloud-init", "status").CombinedOutput()
+	lp, err := exec.LookPath("cloud-init")
+	if err != nil {
```

Comment:
We loose the error details here, should we logger.Debugf() here at least what the error is (or maybe even Noticef)?